package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "lumi-aqara",
   "description": "Control your Xiaomi Smart Home devices with this Lumi Aqara library",
-  "version": "0.0.0-development",
+  "version": "1.7.0",
```

Comment:
Can you delete this change ? The versioning is done automatically by semantic-release. 😉